src/main/resources/application.properties

```diff
@@ -15,4 +15,4 @@ activity.daily-status.repositories.quarkus-extensions-combinations=https://githu
 activity.daily-status.repositories.quarkus-startstop=https://github.com/quarkus-qe/quarkus-startstop/actions/workflows/ci.yaml
 activity.daily-status.repositories.quarkus-test-framework=https://github.com/quarkus-qe/quarkus-test-framework/actions/workflows/daily.yaml
 activity.daily-status.repositories.quarkus-test-suite=https://github.com/quarkus-qe/quarkus-test-suite/actions/workflows/daily.yaml
-activity.daily-status.repositories.quarkus-test-suite-java-17=https://github.com/quarkus-qe/quarkus-jdkspecifics/actions/workflows/daily.yaml
\ No newline at end of file
+activity.daily-status.repositories.quarkus-test-suite-java-17=https://github.com/quarkus-qe/quarkus-jdkspecifics/actions/workflows/ci.yml
```

Comment:
@rsvoboda could you provide context here, please? it sounds awkward that daily status is based on `ci.yml`, what did I miss?

Reply:
ah, I see

Reply:
https://github.com/quarkus-qe/quarkus-jdkspecifics/commit/e0fc4f38501d37f1f3e0997aa49ee01602b40dde

Reply:
See https://github.com/quarkus-qe/quarkus-jdkspecifics/blob/main/.github/workflows/ci.yml#L3-L6

I've unified the definition for GH Actions

Reply:
sure, thx